.github/ISSUE_TEMPLATE/3_Badge_request.yml

```diff
@@ -0,0 +1,55 @@
+name: '💡 Badge Request'
+description: Ideas for new badges
+labels: ['service-badge']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Ideas for new badges
+
+        This issue template is for suggesting new badges which **fetch and display data from an upstream service**. If your suggestion is for a static badge (which shows the same information every time it is requested), it is [already possible to make these](https://github.com/badges/shields/blob/master/doc/static-badges.md). We don't add specific routes for badges which only show static information.
+
+  - type: textarea
+    id: description
+    attributes:
+      label: '📋 Description'
+      description: |
+        A clear and concise description of the new badge.
+
+        - Which service is this badge for e.g: GitHub, Travis CI
+        - What sort of information should this badge show?
+          Provide an example in plain text e.g: "version | v1.01" or as a static badge
+          (static badge generator can be found at https://shields.io/#your-badge )
+    validations:
+      required: true
+
+  - type: textarea
+    id: data
+    attributes:
+      label: '🔗 Data'
+      description: |
+        Where can we get the data from?
+
+        Please consider and cover details like:
+        - Is there a public API?
+        - Does the API require authentication or an API key?
```

Comment:
I would add info that if a private key is required, we will not be able to add this badge. This way, the issue's author can abstain from submitting it, and we save time :)

Reply:
P e r f e c t - this is exactly what I was hoping to see here! Thanks :)